tests/integration/test_retry_aiohttp.py

```diff
@@ -5,17 +5,27 @@
 from uplink.clients import io
 
 from . import test_retry
+import sys
 
 
 @pytest.mark.asyncio
 async def test_retry_with_asyncio(mock_client, mock_response):
-    import asyncio
+    # Check the Python version and define coroutine accordingly
+    if sys.version_info >= (3, 5):
+        # For Python 3.5 and newer
+        async def coroutine():
+            return mock_response
+    elif (3, 3) <= sys.version_info < (3, 5):
```

Comment:
Might be safe to drop this. I don't believe we officially support Python versions below 3.5